static/app/components/events/interfaces/frame/line.tsx

```diff
@@ -440,12 +442,17 @@ const RepeatedFrames = styled('div')`
   display: inline-block;
 `;
 
-const VertCenterWrapper = styled('div')`
+const DefaultLineTitleWrapper = styled('div')`
   display: flex;
   align-items: center;
   justify-content: space-between;
 `;
 
+const VertCenterWrapper = styled('div')`
```

Comment:
there isn't a better name for this is there 😬

Reply:
was already named that, but pushed a change to make it _slightly_ better